src/components/sensor-graph.tsx

```diff
@@ -45,14 +45,17 @@ export default class SensorGraph extends React.Component<SensorGraphProps, Senso
         super(props);
         this.state = {
             xMin: this.props.xStart,
-            xMax: this.props.xEnd
+            xMax: this.props.xEnd,
+            yMin: this.props.preRecording ? this.props.preRecording.min : 0,
+            yMax: this.props.preRecording ? this.props.preRecording.max : 100,
         };
     }
 
     scaleToData() {
         const { sensorRecording, preRecording, prediction } = this.props;
-        let yMin = sensorRecording?.min || preRecording?.min || null;
-        let yMax = sensorRecording?.max || preRecording?.max || null;
+
+        let yMin = sensorRecording?.min || preRecording?.min || 0;
+        let yMax = sensorRecording?.max || preRecording?.max || 100;
```

Comment:
```suggestion
        let yMin = sensorRecording?.min ?? preRecording?.min ?? 0;
        let yMax = sensorRecording?.max ?? preRecording?.max ?? 100;
```
It may not occur in practice, but if there were ever a `sensorRecording.max` or `preRecording.max` of `0` it should be respected.

Reply:
That makes sense, good catch!